tests/python/pytorch/graphbolt/test_csc_sampling_graph.py

```diff
@@ -383,3 +383,117 @@ def test_in_subgraph_heterogeneous():
     assert torch.equal(
         in_subgraph.type_per_edge, torch.LongTensor([2, 2, 1, 3, 1, 3, 3])
     )
+
+
+def check_tensors_on_the_same_shared_memory(t: torch.Tensor, t2: torch.Tensor):
+    """Check if two tensors are on the same shared memory.
+
+    This function copies a random tensor to `t` and checks whether `t2` has the
+    same random value. It cannot use `.data_ptr()` because the memory pointers
+    created by different `shm_open` calls are different.
+    """
+    old_t1 = t.clone()
+    v = torch.randint_like(t, 100)
+    t[:] = v
+    assert torch.equal(t, t2)
+    t[:] = old_t1
```

Comment:
```suggestion
def check_tensors_on_the_same_shared_memory(t1: torch.Tensor, t2: torch.Tensor):
    """Check if two tensors are on the same shared memory.

    This function copies a random tensor to `t1` and checks whether `t2` has the
    same random value. It cannot use `.data_ptr()` because the memory pointers
    created by different `shm_open` calls are different.
    """
    old_t1 = t1.clone()
    v = torch.randint_like(t1, 100)
    t1[:] = v
    assert torch.equal(t1, t2)
    t1[:] = old_t1
```